scripts/code_generation/commit_info_gen/README.md

```diff
@@ -0,0 +1,46 @@
+# commit_info_gen
```

Comment:
Actual documentation is great

Reply:
also on confluence?

Reply:
imo its fine to stay in the repo since this is implementation details, I don't think another team will have to look at this file